templates/web/base/report/updates.html

```diff
@@ -1,11 +1,11 @@
 [% global.last_state = '' %]
-[% prev_state = '' %]
 [% FOREACH update IN updates %]
-[%- IF prev_state == 'hidden' OR prev_state == 'unconfirmed' OR update.problem_state == 'hidden' OR update.problem_state == 'unconfirmed' %]
-  [%- prev_state = update.problem_state %]
+[%- IF global.last_state == 'Hidden' OR global.last_state == 'Unconfirmed' OR update.problem_state == 'hidden' OR update.problem_state == 'unconfirmed' %]
```

Comment:
These will need to be translated strings.

Reply:
👍